src/pe_source/pe_scripts.py

```diff
@@ -88,7 +102,11 @@ def main():
 
     # Set up logging
     logging.basicConfig(
-        format="%(asctime)-15s %(levelname)s %(message)s", level=log_level.upper()
+        filename=CENTRAL_LOGGING_FILE,
+        filemode="a",
+        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+        datefmt="%m/%d/%Y %I:%M:%S",
+        level=log_level.upper(),
```

Comment:
I just noticed two things:
1) You are calling `logging.basicConfig()` twice in this file: above and also [here](https://github.com/cisagov/pe-reports/blob/a1a96655fa2b923e9e630003354e704a2004d758/src/pe_source/pe_scripts.py#L43-L50)).
2) This is the only place where you correctly use the `--log-level` argument (`level=log_level.upper()`).  That argument should be used wherever you call `logging.basicConfig()`, rather than hard-coding it to `"INFO"` everywhere.

Reply:
@dav3r , 
I have updated the config to pull the logging config from the pe_reports module to all of the files except where CLI arguments are taken from the user. I did leave "INFO" hard coded at pe_reports since we aren't taking 'log-level' as an input from the user so there is no need to validate the log level where we pull the basicConfig from pe_reports. See commit [75cfd8d](https://github.com/cisagov/pe-reports/pull/169/commits/75cfd8d9c5f96a9c80ca7cadd57a069abdbf3176)

Reply:
My only issue with that is now you have two loggers in `pe-reports/report_generator.py`:

https://github.com/cisagov/pe-reports/blob/75cfd8d9c5f96a9c80ca7cadd57a069abdbf3176/src/pe_reports/report_generator.py#L36

and

https://github.com/cisagov/pe-reports/blob/75cfd8d9c5f96a9c80ca7cadd57a069abdbf3176/src/pe_reports/report_generator.py#L252-L259

If you have standardized on using the app logger, then there's no need for using the Python standard logger.  It would be nice to get away from hard-coding the logging level with the app logger.

Reply:
@dav3r , 

I have adjusted the config and logger callout and made a change to not have a hard coded log level [b304b6](https://github.com/cisagov/pe-reports/pull/169/commits/b3042b6b6dd8192199b715b808bd05bf27b9c658)

Reply:
Thanks - I still think the logging setup is a bit convoluted and confusing, but it's good enough for me to resolve this conversation.